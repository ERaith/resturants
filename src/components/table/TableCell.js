export function TableCell ({ data }) {
  return (
    <td className="table-cell">
      {data}
    </td>
  )
}