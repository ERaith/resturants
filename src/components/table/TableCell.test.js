import React from "react";
import { TableCell } from "./TableCell";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";

describe("TableCell", () => {
  describe("Snapshots", () => {
    it("should match the snapshot", () => {
      const props = {
        data: "Fake-Data",
      };
      const wrapper = mount(
        <table>
          <tbody>
            <tr>
              <TableCell {...props} />
            </tr>
          </tbody>
        </table>
      );

      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
