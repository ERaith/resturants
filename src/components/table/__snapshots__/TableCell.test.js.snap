// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCell Snapshots should match the snapshot 1`] = `
<table>
  <tbody>
    <tr>
      <TableCell
        data="Fake-Data"
      >
        <td
          className="table-cell"
        >
          Fake-Data
        </td>
      </TableCell>
    </tr>
  </tbody>
</table>
`;
